### DISCUSSION TITLE
何时支持Gitee呢？

### DISCUSSION THREAD

**Discussion by 19980617** [yaolifeng0629/del-repos#3] [Q&A] [answered]
- 功能于 Github 平台相同即可

**Comment by yaolifeng0629** [comment]
@19980617 Corresponding API solutions already exist and will take time to complete.❤️

**Reply by 19980617** [reply, marked as answer]
- Reference #1

**Comment by yaolifeng0629** [comment]
@19980617 the latest v1.0.1 supported
